### PR TITLE
Migrate to gsettings.dart 0.2.0

### PR DESCRIPTION
**Pull request by jpnurmi**

### CODE REVIEW

**Comment by jpnurmi** [comment]
@robert-ancell Is gsettings.dart ready for publishing?

**Comment by robert-ancell** [comment]
> @robert-ancell Is gsettings.dart ready for publishing?

Yes, I think so. I'll update the pub.dev version tomorrow with the Dart version.

**Comment by robert-ancell** [comment]
pub.dev now updates with new implementation (0.2.0)

File: packages/ubuntu_wizard/pubspec.yaml [hunk 1]
```diff
@@ -9,12 +9,15 @@ environment:
...
+  dbus: ^0.5.6
   flutter:
     sdk: flutter
   flutter_localizations:
     sdk: flutter
   form_field_validator: ^1.1.0
-  gsettings: ^0.1.2+1
+  gsettings:
+    git:
+      url: https://github.com/canonical/gsettings.dart.git
```
Comment (robert-ancell) [inline packages/ubuntu_wizard/pubspec.yaml:20]: Can update to `gsettings: ^0.2.0` now.

File: packages/ubuntu_wizard/lib/app.dart [hunk 2]
```diff
@@ -27,7 +27,7 @@ Future<void> runWizardApp(
   List<String>? serverArgs,
   List<SingleChildWidget>? providers,
 }) async {
-  final interfaceSettings = GSettings(schemaId: 'org.gnome.desktop.interface');
+  final interfaceSettings = GSettings('org.gnome.desktop.interface');
```
Comment (robert-ancell) [inline packages/ubuntu_wizard/lib/app.dart:30]: This needs to be closed at some point with `await interfaceSettings.close()`

**Comment by oSoMoN** [comment]
LGTM, thanks!

**Pull request merged**